package.json

```diff
@@ -43,7 +43,7 @@
   },
   "devDependencies": {
     "asar": "^2.0.1",
-    "electron-builder": "22.10.5",
+    "electron-builder": "22.11.4",
     "electron-builder-squirrel-windows": "22.10.5",
```

Comment:
We try to keep `electron-builder-squirrel-windows` on the same version since they work together, so let's update that one to match.